Final_Project/Project_Proposal.Rmd

```diff
@@ -0,0 +1,152 @@
+
+### Link 1
+[Interactive Map Link](https://pad.human.cornell.edu/profiles/index.cfm)
+
+Link 1 leads to an example of an interactive map of NYS County polygons which contain demographic, social and economic data, courtesy of Cornell University. This example shows one of the goals of this project. I hope to have an interactive map of counties that contains the summary data for the tick pathogen prevalence data as well as the land use / land cover data. Additionally, the time-series charts (see image 3) for each county should also appear. Ideally, when clicking on a county, these features should either pop up or open in a new link, as this example from Cornell does. 
+
+## Proposed Data Sources:
+There are three data sources required for this project:
+
+1. Tick pathogen prevalence data
+2. Land use / land cover data
+3. Spatial polygon data of NYS Counties
+
+### Tick pathogen prevalence data
+The New York State Department of Health (NYSDOH) Vector Ecology Lab conducts seasonal tick-surveys throughout NYS. These surveys measure density of ticks in the environment, and also measure the pathogen prevalence of <i>B. burgdorferi</i>, <i>A. phagocytophilum</i>, <i>B. microti</i>, and <i>B. miyamotoi</i> in the collected ticks. Pathogen prevalence is determined through PCR testing of individual tick specimens. These data are available to the public via [Health Data NY](https://health.data.ny.gov/), for both [adult](https://health.data.ny.gov/Health/Access-Adult-Deer-Tick-Collection-Data-by-County-E/fkdr-6a5t) and [nymphal](https://health.data.ny.gov/Health/Access-Nymph-Deer-Tick-Collection-Data-by-County-E/7qid-kum3) tick density. These data on Health Data NY show tables and maps, but do not conduct any statistical testing or plot any time-series charts. They also do not utilize and ecological data, as is the goal of this project. This project will only be looking at adult tick data, as there are generally more adult tick samples from collections.
+
+### Land use / land cover data
+Quality raster data for land use and/or land cover type is available from different sources. For this project, I plan to use the National Land Cover Dataset (NLCD) 2016 (CONUS). The NLCD 2016 is available for download from the [Multi-Resolution Land Characteristics Consortium (MRLC) website](https://www.mrlc.gov/data/nlcd-2016-land-cover-conus), which describes the dataset as a product from the U.S. Geological Survey and several other federal agencies. The data is gathered via a multitude of methods, listed in the description at the link above. The dataset is a raster dataset which breaks down each pixel of the contiguous United States into the specified categories shown below:
```

Comment:
you could also use MODIS-derived landsat which is 1km in resolution.  The 30m NLCD might be overkill since you are going to aggregate it anyways. We'll actually cover this later in this course.

Reply:
see https://rdrr.io/cran/MODIS/man/getProduct.html